years.sh

```diff
@@ -11,6 +12,7 @@ if (grep -r -L "Copyright \+(c) \+2009-.*rultor.com" \
   --include=*.java --include=*.xml --include=*.vm --include=*.groovy \
   --include=*.txt --include=*.fml --include=*.properties} \
   --exclude-dir "target" --exclude-dir "apache-maven-*" . \
+  --exclude-dir ".idea" \
```

Comment:
@mkordas ignore these two lines, it's a Github/Git issue. They are the same as in master but resulted from a 3-way merge in the branch and somehow it doesn't get this right when diffing :)